src/components/app-content-renderer/second-panel.js

```diff
@@ -22,7 +22,20 @@ import RecentlyVisited from '../recently-visited/recently-visited';
 import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
 
 const SecondPanel = () => {
-  const { isProd, isBeta } = useChrome();
+  const {
+    isProd,
+    isBeta,
+    visibilityFunctions: { apiRequest },
+  } = useChrome();
+  const [quayVisible, setQuayVisible] = useState(false);
+
+  useEffect(() => {
+    apiRequest({
+      url: `/${isBeta() ? 'beta' : ''}/apps/quay/plugin-manifest.json`,
+      matcher: 'isNotEmpty',
+      accessor: 'version',
+    }).then((result) => setQuayVisible(result));
```

Comment:
I know it's kind of a quick hack, but can we import the actual valculate visibility function in here? Doesn't have to be for this specific PR.

Reply:
It will go away once Quay is fully onboarded. I just did not want to make use of the usual prod/beta flags because we would have to remove them again.